### PR TITLE
Bill's Actually Discount Guns and FS Vendor Expansion

### PR DESCRIPTION
**Pull request by ValoTheValo**
## About The Pull Request
This PR adds a few more weapons to Billomat and FS Vendors, aswell as tweaks the prices of them to be more reasonable (made with billomat creators blessing and input)
In Depth Changes
Havelock (700$ to 500$), Spider Rose(2500$ to 1600$), MK58(1500$ to 600$) and Cog(2000 to 1350$) Prices were lowered as the performance of the weapons was not worth the high asking price especially when compared to similar weapons

Regulator Price was Increased from 1500 to 2000 as the regulator is probably the best pump shotgun in the game and a price that was lower than the Gladstone didn't seem right (also requested by shown)

Vending Inventory Edits
Gladstone and Molly were moved back to frozen star vendor as they are frozen star weapons and used to be in it
Spider Rose was added to the frozen star vendor
Regulator was added to the Billomat Vendor

## Why It's Good For The Game
Many people complained that a few of the weapons in the billomat machine were extremely overpriced and that the FS machine lacked variety. I have fixed these issues and this will hopefully have a wider variety of guns showing up in game.
## Changelog
:cl:
tweak: Gladstone, Molly and Spider-Rose added to FS vendor
tweak: Regulator added to Maint Vendor
balance: MK58,Havelock,Cog and Regulator prices Adjusted
:cl:

### CODE REVIEW

**Comment by ValoTheValo** [comment]
ignore the wierd first few commits i have 0 clue why they are showing up, they dont impact anything so it should be fine

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -996,9 +996,12 @@
 					/obj/item/weapon/gun/projectile/olivaw = 5,
 					/obj/item/weapon/gun/projectile/giskard = 5,
 					/obj/item/weapon/gun/energy/gun/martin = 5,
+					/obj/item/weapon/gun/energy/gun = 3
```
Comment (Firefox13) [inline code/game/machinery/vending.dm:999]: bruh

Comment (ValoTheValo) [reply code/game/machinery/vending.dm:999]: fuck i forgot the comma

Comment (Firefox13) [reply code/game/machinery/vending.dm:999]: you fucking put a parent into the list
go test it or something

Comment (ValoTheValo) [reply code/game/machinery/vending.dm]: parent is the spider rose for some reason and martin and mounted are variations on that

Comment (Firefox13) [reply code/game/machinery/vending.dm]: bruh

**Comment by zombiemimic** [comment]
Maybe add a markup on the FS guns

File: code/game/machinery/vending.dm (hunk 1 above)
Comment (ValoTheValo) [reply code/game/machinery/vending.dm]: > you fucking put a parent into the list
> go test it or something

shitcode
i have tested it and it works fine

**Comment by Jan-Blasiak** [comment]
> Maybe add a markup on the FS guns

This is CEV Eris, always add markup

**Comment by TheShown911** [comment]
Guns on the bill vendor should be overpriced.

File: code/modules/projectiles/guns/projectile/pistol/mk58.dm [hunk 2]
```diff
@@ -6,7 +6,7 @@
 	item_state = "pistol"
 	origin_tech = list(TECH_COMBAT = 2, TECH_MATERIAL = 2)
 	matter = list(MATERIAL_PLASTEEL = 12, MATERIAL_PLASTIC = 6)
-	price_tag = 1400
+	price_tag = 400
```
Comment (Firefox13) [inline code/modules/projectiles/guns/projectile/pistol/mk58.dm:9]: Did you tested the prices? How heavily it affects the spawn rate in maints? This(and the others) pistol might start appearing literally everywhere with such low prices

**Comment by Firefox13** [comment]
> 
> 
> Guns on the bill vendor should be overpriced.

yes
feature

File: code/modules/projectiles/guns/projectile/pistol/mk58.dm (hunk 2 above)
Comment (L74814) [reply code/modules/projectiles/guns/projectile/pistol/mk58.dm:9]: > Did you tested the prices? How heavily it affects the spawn rate in maints? This(and the others) pistol might start appearing literally everywhere with such low prices

image

so probably rarity changes would be required to keep the actual spawn ratios I guess

Comment (ValoTheValo) [reply code/modules/projectiles/guns/projectile/pistol/mk58.dm:9]: didn't realize that, ill check how often the edited guns spawn

**Review by drexample — CHANGES_REQUESTED** [review]
Pls increase prices. Money need more uses and this is a good one.

**Comment by ValoTheValo** [comment]
> Pls increase prices. Money need more uses and this is a good one.

This good enough? raised price on mk58 by 100 and Havelock by 150? or do you mean with other weapons?

**Comment by ValoTheValo** [comment]
execution and balance was kinda fucked on this one

**Pull request closed without merging**